packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1374,7 +1374,7 @@ async function handler(
             }
           }
 
-          if (noEmail !== true) {
+          if (noEmail !== true && (!requiresConfirmation || isOrganizerRescheduling)) {
```

Comment:
!requiresConfirmation || isOrganizerRescheduling

- We send the rescheduled email If the event doesn't require confirmation OR organiser is rescheduling the meeting